### ISSUE TITLE
Spatial Structure: Export from PKSim:

### ISSUE THREAD

**Issue by msevestre** [Open-Systems-Pharmacology/MoBi#897]
>Parameters that are defined in the Individuals-BB be populated with "NaN"-values in the SS-BB of PK-Sim modules.

@PavelBal @Yuri05 Just to clarify. Does this mean that a parameter like Volume or Q_spec will have no value?

**Comment by PavelBal** [comment]
Yes. When viewing a PK-Sim module in MoBi, the user will select an Individuals-BB and the shown values will correspond to those from the individual.

When converting the PK-Sim module to an extension module, the used must select an Individual-BB and the values will be applied.

**Comment by msevestre** [comment]
Can you clarify how this is supposed to happen.
When I select the spatial structure, I also have to select and individual???
Or is this a special mode to see the values?

What do you mean with the lastest comment? The value will be applied? In this case, the values are now in the spatial structure? Why the difference of behaviour?

**Comment by PavelBal** [comment]
Meeting 22.02.2023: Not required for the first version.

Keep in specs and decide if this should be implemented later.

**Comment by PavelBal** [comment]
Now this task: https://github.com/Open-Systems-Pharmacology/MoBi/issues/947
